index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mansa Susa</title>
+    <link rel="stylesheet" href="styles.css" />
+</head>
+<body>
+  <header>
+      <h3 id="logo">Mansa Susa</h3>
```

Comment:
- [ ] Please change `h3`  with  `a` tag on this logo.

Reply:
i have replaced the h3 with the a tag